template-sample-app/template.json

```diff
@@ -42,33 +44,25 @@
       "react-map-gl": "^5.3.0",
       "react-redux": "^7.2.2",
       "react-router-dom": "^6.0.0-beta.0",
-      "react-scripts": "4.0.1"
+      "react-scripts": "4.0.1",
+      "start-server-and-test": "^1.12.0"
     },
     "scripts": {
-      "start": "react-app-rewired start",
-      "build": "react-app-rewired build",
-      "test": "react-app-rewired test",
-      "lint": "eslint './src/**/*.{js,jsx}'",
-      "lint:fix": "eslint './src/**/*.{js,jsx}' --fix",
-      "format": "prettier --write \"**/*.+(js|jsx|json|css|md)\" --ignore-path .gitignore",
+      "cypress": "npx cypress open",
+      "cy:run": "npx cypress run",
+      "cy:ci": "START_SERVER_AND_TEST_INSECURE=1 start-server-and-test start https-get://localhost:3000 cy:run",
```

Comment:
I also think the CI for cypress can be a bit 'too much' for a basic setup

Reply:
I think so